# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../spicctrl/install-prefix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur spicctrl-1.8-orig/Makefile spicctrl-1.8/Makefile
--- spicctrl-1.9-orig/Makefile	2005-02-10 17:21:47.000000000 +0200
+++ spicctrl-1.9/Makefile	2005-05-28 18:35:46.718907544 +0300
@@ -6,7 +6,7 @@
 LIBS =
 DOCS = AUTHORS CHANGES LICENSE
 VERSION = 1.9
-PREFIX = /usr/local
+PREFIX = /usr
 DESTBIN = $(PREFIX)/bin
 DESTMAN = $(PREFIX)/man/man1
 DESTDOC = $(PREFIX)/share/doc/spicctrl-$(VERSION)
